doc-source-md/development-frameworks/mapbox.md

```diff
@@ -0,0 +1,350 @@
+# Using MAPCAT vector based map tiles with Mapbox GL JS
+
+You can use [MAPCAT](http://mapcat.com) services on your website with our [Mapbox GL JS](https://www.npmjs.com/package/@mapcat/mapbox-gl) fork which offers you a JavaScript library that renders interactive maps from vector tiles using WebGL. This fork is able to use multilanguage (i18n) support over [Mapbox GL JS](https://www.mapbox.com). 
+We intend to periodically update this fork from the original Mapbox GL JS repository, so if you don't want to use our multilanguage feature you can use original [Mapbox GL JS](https://www.mapbox.com).
```

Comment:
Inkább így írnám:
...repository. If you don't want to use our multilanguage feature, you can use the original...